step-functions-data-science-sdk/step_functions_mlworkflow_processing/step_functions_mlworkflow_scikit_learn_data_processing_and_model_evaluation.ipynb

```diff
@@ -0,0 +1,987 @@
+    "                \"glue:GetJobRuns\",\n",
+    "                \"glue:StartJobRun\",\n",
+    "                \"lambda:InvokeFunction\",\n",
+    "                \"sagemaker:CreateTransformJob\",\n",
+    "                \"sagemaker:CreateEndpoint\",\n",
+    "                \"sagemaker:CreateEndpointConfig\",\n",
+    "                \"sagemaker:CreateHyperParameterTuningJob\",\n",
+    "                \"sagemaker:CreateModel\",\n",
+    "                \"sagemaker:CreateProcessingJob\",\n",
+    "                \"sagemaker:CreateTrainingJob\",\n",
+    "                \"sagemaker:DeleteEndpoint\",\n",
+    "                \"sagemaker:DeleteEndpointConfig\",\n",
+    "                \"sagemaker:DescribeHyperParameterTuningJob\",\n",
+    "                \"sagemaker:DescribeProcessingJob\",\n",
+    "                \"sagemaker:DescribeTrainingJob\",\n",
+    "                \"sagemaker:DescribeTransformJob\",\n",
+    "                \"sagemaker:ListProcessingJobs\",\n",
+    "                \"sagemaker:ListTags\",\n",
+    "                \"sagemaker:StopHyperParameterTuningJob\",\n",
+    "                \"sagemaker:StopProcessingJob\",\n",
+    "                \"sagemaker:StopTrainingJob\",\n",
+    "                \"sagemaker:StopTransformJob\",\n",
+    "                \"sagemaker:UpdateEndpoint\",\n",
+    "                \"sns:Publish\",\n",
+    "                \"sqs:SendMessage\"\n",
+    "            ],\n",
+    "            \"Resource\": \"*\"\n",
+    "        }\n",
+    "    ]\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "3. Replace **NOTEBOOK_ROLE_ARN** with the ARN for your notebook that you created in the previous step in the above Policy.\n",
+    "4. Choose **Review policy** and give the policy a name such as `StepFunctionsWorkflowExecutionPolicy`.\n",
+    "5. Choose **Create policy**.\n",
+    "6. Select **Roles** and search for your `StepFunctionsWorkflowExecutionRole` role.\n",
+    "7. Under the **Permissions** tab, click **Attach policies**.\n",
+    "8. Search for your newly created `StepFunctionsWorkflowExecutionPolicy` policy and select the check box next to it.\n",
+    "9. Choose **Attach policy**. You will then be redirected to the details page for the role.\n",
+    "10. Copy the StepFunctionsWorkflowExecutionRole **Role ARN** at the top of the Summary."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# paste the StepFunctionsWorkflowExecutionRole ARN from above\n",
+    "workflow_execution_role = \"\" "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create StepFunctions Workflow execution Input schema"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate unique names for Pre-Processing Job, Training Job, and Model Evaluation Job for the Step Functions Workflow\n",
```

Comment:
`@ignorable`: should this be in markdown prior to the code cell?

Reply:
Will be happy to incorporate the changes and resolve them after the initial merge so that we can unblock the launch.